lib/Cro/HTTP/Client.pm6

```diff
@@ -119,6 +119,7 @@ class Cro::HTTP::Client::Policy::Timeout does Cro::Policy::Timeout[%(
 #| for multiple requests, as well as allowing configuration of common properties of
 #| many requests at construction time.
 class Cro::HTTP::Client {
+    my class PipelineClosedBeforeHeaders is Exception { }
```

Comment:
I think the pattern in jnthn work is to declare exception with X:: at top of the file and have a message for when it throws.

If I misunderstand and it fits better here (maybe this Exception never throws), please add newline